src/api/app/assets/stylesheets/webui2/breadcrumbs-component.scss

```diff
@@ -0,0 +1,3 @@
+.breadcrumb-item > a {
+    color: #6c757d;
```

Comment:
Why not one of the grays?

Reply:
This gray is what we have in the "old" breadcrumb. I just copied it.

Reply:
I don't think it's worth it.

Reply:
Ok, I will try to use one close to that